pkg/generator/keytool.go

```diff
@@ -25,6 +27,133 @@ func init() {
 	tempDir = dir
 }
 
+type cmdRunner func(cmdName string, args []string) ([]byte, error)
+
+func baseCommand(execCmd string, baseArgs []string) func(cmdName string, args []string) ([]byte, error) {
+	return func(cmdName string, args []string) ([]byte, error) {
+		cmdArgs := []string{}
+		cmdArgs = append(cmdArgs, cmdName)
+		cmdArgs = append(cmdArgs, baseArgs...)
+		cmdArgs = append(cmdArgs, args...)
+		cmd := exec.Command(execCmd, cmdArgs...)
+		return cmd.CombinedOutput()
+	}
+}
+
+// AliasMgr an interface for managing keytool aliases
+type AliasMgr interface {
+	References() ([]string, []string)
+	LoadReferenceData(data map[string][]byte) error
+	Generate() error
+}
+
+// NewKeyTool creates new keytool instance
+func NewKeyTool(key *v1alpha1.KeyConfig) *KeyTool {
```

Comment:
I think this should return (*KeyTool, error) to keep things consistent with other generators
```suggestion
func NewKeyTool(key *v1alpha1.KeyConfig) (*KeyTool, error) {
```